pandas/tests/indexes/multi/test_conversion.py

```diff
@@ -82,6 +82,29 @@ def test_to_frame():
     tm.assert_frame_equal(result, expected)
 
 
+def test_to_frame_dtype_fidelity():
```

Comment:
can you add the issue number (ideally add to most of the tests you added, where appropriate)